rust/parser/typeql.pest

```diff
@@ -551,30 +560,31 @@ CURLY_CLOSE = _{ "}" }
 COMMA = _{ "," }
 SEMICOLON = _{ ";" }
 COLON = _{ ":" }
+DOT = _{ "." }
 DOUBLE_DOT = _{ ".." }
 QUESTION = _{ "?" }
 ARROW = _{ "->" }
 
 // FRAGMENTS OF KEYWORDS =======================================================
 
 IDENTIFIER_CHAR = @{ ASCII_ALPHA
-                   | '\u{00C0}'..'\u{00D6}'
-                   | '\u{00D8}'..'\u{00F6}'
-                   | '\u{00F8}'..'\u{02FF}'
-                   | '\u{0370}'..'\u{037D}'
-                   | '\u{037F}'..'\u{1FFF}'
-                   | '\u{200C}'..'\u{200D}'
-                   | '\u{2070}'..'\u{218F}'
-                   | '\u{2C00}'..'\u{2FEF}'
-                   | '\u{3001}'..'\u{D7FF}'
-                   | '\u{F900}'..'\u{FDCF}'
-                   | '\u{FDF0}'..'\u{FFFD}'
+                   // | '\u{00C0}'..'\u{00D6}'
```

Comment:
Why are these commented?

Reply:
That was not meant to be pushed, fixed.
That was for the autogenerated tests, so that the printed generated queries which failed would be somewhat readable.